plugin/error_vis/popups.py

```diff
@@ -45,7 +45,7 @@
 {content}
 """
 
-FULL_DOC_TEMPLATE = """### Full doxygen comment:
+FULL_DOC_TEMPLATE = """### Detailed documentation:
```

Comment:
This right here is an aesthetic change, which you are free to keep or throw out - I think it is more consistent since it is more similar to the title of the other related section, "Brief documentation".